packages/react-urql/examples/1-getting-started/src/pages/Home.tsx

```diff
@@ -13,16 +13,16 @@ interface QueryResponse {
   todos: ITodo[];
 }
 
-const ToggleTodo = `
-  mutation($id: ID!) {
+const ToggleTodo = gql`
```

Comment:
is this gql call needed, I removed them because it shows we don't force you to use it unless you want fragments (said in docs)

Reply:
Oh I see, it's not needed no. But the types also make it clear that strings can be provided no?